AutoRest/Generators/NodeJS/NodeJS/NodeJSCodeGenerator.cs

```diff
@@ -16,7 +16,7 @@ namespace Microsoft.Rest.Generator.NodeJS
     public class NodeJSCodeGenerator : CodeGenerator
     {
         private const string ClientRuntimePackage = "ms-rest version 1.2.0";
-        private const bool DisableTypeScriptGeneration = false;    // Change to true if you want to no longer generate the 3 d.ts files, for some reason
+        public const bool DisableTypeScriptGeneration = false;    // Change to true if you want to no longer generate the 3 d.ts files, for some reason
```

Comment:
Should this be exposed as a setting? If so, change this to a property and add an attribute:

```csharp
[SettingsInfo("Disables TypeScript generation.")]
public bool DisableTypeScriptGeneration {get;set;}
```